salt/modules/debbuild.py

```diff
@@ -452,20 +455,13 @@ def build(runas,
     return ret
 
 
-def make_repo(repodir,
-              keyid=None,
-              env=None,
-              use_passphrase=False,
-              gnupghome='/etc/salt/gpgkeys',
-              runas='root'):
+def make_repo(repodir, keyid=None, env=None, use_passphrase=False, gnupghome='/etc/salt/gpgkeys', runas='root'):
```

Comment:
Please adhere to the style guide reference regarding our preference for vertical layout: https://docs.saltstack.com/en/latest/topics/development/conventions/style.html#vertical-is-better